src/back/routes/notification.ts

```diff
@@ -30,6 +33,17 @@ async function sendNotification(req: WithAuth) {
     throw new RequestError('Invalid data', RequestError.BadRequest)
   }
 
+  const users = await UserModel.getDiscordIdsByAddresses(isArray(recipient) ? recipient : [recipient])
+
+  for (const user of users) {
+    await DiscordService.sendDirectMessage(user.discord_id, {
```

Comment:
Should we try catch this? If one the Discord messages fails then the Push notification will fail.

Reply:
`sendDirectMessage` it's not async, so there's no need to trycatch it

Reply:
Oh so you need to remove the await here 😄